pkg/common/common.go

```diff
@@ -116,3 +120,10 @@ func GetMonitoringData(data string) (string, error) {
 
 	return encodedData, nil
 }
+
+func IsMonitoringEnabled(dscMonitoring *dsciv1.Monitoring) bool {
+	if dscMonitoring != nil && dscMonitoring.ManagementState == operatorv1.Managed {
```

Comment:
It's `return dscMonitoring != nil && dscMonitoring.ManagementState == operatorv1.Managed`
(probably shorter parameter name works better for one line)

Reply:
you mean something like:
```
func IsMonitoringEnabled(monitoring *dsciv1.Monitoring) bool {
	if monitoring != nil && monitoring.ManagementState == operatorv1.Managed {

```
?

Reply:
Like this
```
func IsMonitoringEnabled(m *dsciv1.Monitoring) bool {
	return m != nil && m.ManagementState == operatorv1.Managed 
}
```
since for such a short code it's obvious from the declaration what `m` means.
But I do not insist.

Reply:
updated